packages/react/src/components/TileCatalog/TileCatalog.jsx

```diff
@@ -23,9 +23,17 @@ export const propTypes = {
     pageSize: PropTypes.number,
     pageText: PropTypes.string,
     /** Gets called back with arguments (page, maxPage) */
-    pageOfPagesText: PropTypes.func,
-    nextPageText: PropTypes.string,
-    prevPageText: PropTypes.string,
+    pageOfPagesText: deprecate(
+      PropTypes.func,
+      `The prop \`pageOfPagesText\` has been deprecated for the \`SimplePagination\` component. It will be removed in the next major release. Please use \`i18n.pageOfPagesText\` instead.`
+    ),
+    nextPageText: deprecateString(),
+    prevPageText: deprecateString(),
+    i18n: PropTypes.shape({
+      pageOfPagesText: PropTypes.func,
```

Comment:
For our new strings are we removing these functions? I have not really thought out the repercussions of that but I do know it's something that we have been requested to do.

Reply:
I assumed that would be tackled in a separate issue.